src/main/java/net/sf/jabref/logic/help/HelpFile.java

```diff
@@ -48,7 +48,8 @@
     FETCHER_SCIENCEDIRECT(""),
     FETCHER_BIBSONOMY_SCRAPER(""),
     DATABASE_PROPERTIES("DatabaseProperties"),
-    FIND_DUPLICATES("FindDuplicates");
+    FIND_DUPLICATES("FindDuplicates"),
+    MIGRATE_OLD_DATABASE("migrateOldDatabase");
```

Comment:
Could you rename it here to migratePre3dot6Database, too? See https://github.com/JabRef/help.jabref.org/pull/56/ :innocent: